dll/opengl/opengl32/swimpl.c

```diff
@@ -1241,10 +1241,10 @@ static void read_color_span_##__bpp(GLcontext* ctx,
     {                                                                           \
         Buffer -= __pixel_size;                                                 \
         UNPACK_COLOR_##__bpp(GET_PIXEL_##__bpp((__type*)Buffer),                \
-                &blue[n], &green[n], &red[n]);                                  \
+                &red[n], &green[n], &blue[n]);                                  \
         alpha[n] = 0;                                                           \
     }                                                                           \
-}
+} // CORE-16221 - Fixed by swap at Line #1244.
```

Comment:
Interesting... Doesn't the one at line 1297 also needs fixing?

Reply:
Yes, it does.

Reply:
@Doug-Lyons can you remove the comment about fixing a bug ? That is what git history is for :-)